app.rb

```diff
@@ -1,20 +1,236 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+ruby_apartments = []
+ruby_tenants = []
+
+ruby_apartments = apartments.map {|ap| Apartment.new(ap, tenants)}
```

Comment:
I thought you were doing this all wrong, then i looked at your initialize method! Really cool.